session/cookie.js

```diff
@@ -26,7 +26,11 @@ var Cookie = module.exports = function Cookie(options) {
   this.path = '/';
   this.maxAge = null;
   this.httpOnly = true;
-  if (options) merge(this, options);
+  if (options && typeof options == 'object') {
+    merge(this, options);
+  } else {
+    throw new Error('options must be a object');
```

Comment:
This should probably be a TypeError object

Reply:
yeah